lib/messagebird.js

```diff
@@ -500,6 +523,17 @@ module.exports = function (accessKey, timeout, features) {
         };
 
         httpRequest({ method: 'GET', path: '/verify/' + id, params: params }, callback);
+      },
+
+      /**
+       * Get email message details
+       *
+       * @param {String} email message id
+       * @param {Function} callback
+       * @return {void}
+       */
+      getVerifyEmailMessage: function (id, callback) {
```

Comment:
Also would be really nice to add simple tests for new methods 🙏

Reply:
I have added the tests